src/xbot/common/controls/sensors/XGamepad.java

```diff
@@ -1,10 +1,33 @@
 package xbot.common.controls.sensors;
 
+
 public interface XGamepad {
 
-    public XJoystick getLeftStick();
-    public XJoystick getRightStick();
-    public XJoystick getDpad();
-    public XJoystick getLeftTrigger();
-    public XJoystick getRightTrigger();
+    //double
+    //Joysticks
+    double getXStickLeft();
+    double getXStickRight();
+    double getYStickLeft();
+    double getYStickRight();
+    
+    //Triggers
+    double getLeftTriggerAxis();
```

Comment:
It does seem reasonable to try and indicate with the name that the trigger isn't just another button though?